src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,47 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, S> {
+    private T first;
+    private S second;
 
+    private Pair(T first, S second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public T getFirst() {
+        return first;
+    }
+
+    public S getSecond() {
+        return second;
+    }
+
+    static <T, S> Pair of(T t, S s) {
```

Comment:
do we need here default access modifier?

Reply:
changed default access modifier to public